SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/IIngestion.cs

```diff
@@ -1,18 +1,30 @@
 // Copyright (c) Microsoft Corporation.  All rights reserved.
 
-using Microsoft.AppCenter.Ingestion.Models;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using System;
+using System.Collections.Generic;
+using Microsoft.AppCenter.Ingestion.Models;
 
 namespace Microsoft.AppCenter.Ingestion
 {
+    /// <summary>
+    /// Interface to send logs to the Ingestion service.
+    /// </summary>
     public interface IIngestion : IDisposable
     {
-        IServiceCall PrepareServiceCall(string appSecret, Guid installId, IList<Log> logs);
-        Task ExecuteCallAsync(IServiceCall call);
-        void Close();
+        /// <summary>
+        /// Update log URL.
+        /// </summary>
+        /// <param name="logUrl"></param>
         void SetLogUrl(string logUrl);
+
+        /// <summary>
+        /// Send logs to the Ingestion service.
+        /// </summary>
+        /// <param name="appSecret">A unique and secret key used to identify the application</param>
+        /// <param name="installId">Install identifier</param>
+        /// <param name="logs">Payload</param>
+        IServiceCall Call(string appSecret, Guid installId, IList<Log> logs);
```

Comment:
`IServiceCall` with callback isn't awaiting `System.Task`, but essentially it is the same